### PR TITLE
add epb_queue option to packet

### PR DESCRIPTION
**Pull request by sensokame**
add queue option to packet in read and write operations.

### CODE REVIEW

**Comment by sensokame** [comment]
@kayoub5 can you check this PR?

File: include/light_pcapng.h [hunk 1]
```diff
@@ -47,6 +47,7 @@ extern "C" {
 
 #define LIGHT_OPTION_EPB_FLAGS             2
 #define LIGHT_OPTION_EPB_DROPCOUNT         4
+#define LIGHT_OPTION_EPB_QUEUE			   6
```
Comment (kayoub5) [inline include/light_pcapng.h:50]: Inconsistent spacing

File: src/light_pcapng_ext.c [hunk 2]
```diff
@@ -517,6 +525,11 @@ int light_write_packet(light_pcapng pcapng, const light_packet_interface* packet
 		light_option dropcount_opt = light_create_option(LIGHT_OPTION_EPB_DROPCOUNT, 8, &packet_header->dropcount);
 		light_add_option(NULL, packet_block_pcapng, dropcount_opt, false);
 	}
+	if (packet_header->queue)
```
Comment (kayoub5) [inline src/light_pcapng_ext.c:528]: 0 is a valid queue value or not?

File: tests/test_write_pcapng.c [hunk 3]
```diff
@@ -53,6 +53,7 @@ int main(int argc, const char** args) {
 	pkt_header.original_length = 1024;
 	pkt_header.flags = 0x1; // direction indicator
 	pkt_header.dropcount = 0;
+	pkt_header.queue = 1;
```
Comment (kayoub5) [inline tests/test_write_pcapng.c:56]: No updates to results files?

File: src/light_pcapng_ext.c (hunk 2 above)
Comment (sensokame) [reply src/light_pcapng_ext.c:528]: I think it's the default value. So no queue means it's 0

File: tests/test_write_pcapng.c (hunk 3 above)
Comment (sensokame) [reply tests/test_write_pcapng.c:56]: There is one updated test file.

**Comment by sensokame** [comment]
@kayoub5 can you check the PR again?

**Review by kayoub5 — APPROVED** [review]

**Comment by kayoub5** [comment]
@sensokame please rebase your branch

**Comment by sensokame** [comment]
@kayoub5 done, can you check the branch now?

**Pull request merged**